modules/complex-number/test/test_baldin_alexey_complex_number.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Baldin Alexey
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Baldin_Alexey_ComplexNumberTest, Can_Create_Zero) {
+    // Arrange
+    double re = 0.0;
+    double im = 0.0;
+
+    // Act
+    ComplexNumber z(re, im);
+
+    // Assert
+    EXPECT_EQ(re, z.getRe());
+    EXPECT_EQ(im, z.getIm());
+}
+
+TEST(Baldin_Alexey_ComplexNumberTest, Multiple_By_Zero) {
+    // Arrange
+    double re = 123.0;
+    double im = 456.0;
+    ComplexNumber z(re, im);
+    ComplexNumber zero(0.0, 0.0);
+
+    // Act
+    z = z * zero;
+
+    // Assert
+    EXPECT_EQ(z.getRe(), 0.0);
+    EXPECT_EQ(z.getIm(), 0.0);
+}
+
+TEST(Baldin_Alexey_ComplexNumberTest, Negative_Number_Less_Then_Positive) {
```

Comment:
Исправьте опечатку в названии теста
```suggestion
TEST(Baldin_Alexey_ComplexNumberTest, Negative_Number_Less_Than_Positive) {
```

Reply:
Исправлено